misc/CCSynth/CC/DataInsights/tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+envlist = py36,py35,py37
+
+[testenv]
+whitelist_externals = powershell
```

Comment:
we should not use "whitelist" and "blacklist". Maybe just remove this file since it was only being used for testing.